### PR TITLE
Add main API for initialization and destroy

### PR DESCRIPTION
**Pull request by Zmoki**
@nd0ut @bautrukevich what do you think about that API and naming?

### CODE REVIEW

File: .size-snapshot.json [hunk 1]
```diff
@@ -1,26 +1,50 @@
...
   },
   "dist/uploadcare-uploader.js": {
-    "bundled": 308,
-    "minified": 114,
-    "gzipped": 110
+    "bundled": 4921,
+    "minified": 2267,
+    "gzipped": 995
+  },
+  "dist/Uploader-uploader.esm.js": {
```
Comment (nd0ut) [inline .size-snapshot.json]: I didn't find such bundle in the rollup config. Probably we need to regenerate this snapshot?

File: README.md [hunk 2]
```diff
@@ -1 +1,17 @@
...
+
+```javascript
+import uploadcare from '@uploadcare/uploader'
+
+uploadcare.initialize(container) // Array<Uploader>
+uploadcare.deinitialize(container) // boolean
+
+uploadcare.createUploader(target, settings) // Uploader
+uploadcare.destroyUploader(uploader) // boolean
+uploadcare.findUploaders(container) // Widget
```
Comment (nd0ut) [inline README.md]: Why `Widget` and not `Array<Uploader>` ?

File: README.md [hunk 3]
```diff
@@ -1 +1,17 @@
 # uploadcare-uploader
+
+```javascript
+import uploadcare from '@uploadcare/uploader'
+
+uploadcare.initialize(container) // Array<Uploader>
```
Comment (nd0ut) [inline README.md]: Maybe use `attach / detach` or `watch / unwatch` (if we'll track container children)?
`initialize / deinitialize` sounds to me like I should call them anyway even if I create uploader manually.

File: .size-snapshot.json (hunk 1 above)
Comment (Zmoki) [reply .size-snapshot.json]: oops

File: README.md (hunk 2 above)
Comment (Zmoki) [reply README.md]: mistake, thank you

**Pull request merged**